src/plugins/discover/public/application/components/default_discover_table/_doc_table.scss

```diff
@@ -69,6 +69,23 @@ doc-table {
   }
 }
 
+.osdDiscoverTable {
+  @include ouiCodeFont;
+
+  & > tbody > tr:not(.osdDiscoverInViewRow):not(.osdDiscoverExpandedRow) {
+    visibility: hidden;
+  }
+
+  .osdDiscoverExpandedRow {
+    background: $euiColorEmptyShade;
+    border-top: none !important;
+
+    & > td {
+      padding: 2em;
```

Comment:
I know this makes the syle similar tot he old one, but given that its reducing space for the user, can we skip this?

Reply:
The UI doesn't have a background and without this, there is no visual boundary of where what is. I feel this is not too much price for have a good-ish UX.

Reply:
Have you rebased? We have a background and borders now :)

Reply:
oh nice. i will.

Reply:
dropped